### ISSUE TITLE
CF history link

### ISSUE THREAD

**Issue by taylor13** [cf-convention/cf-convention.github.io#570]
Hi all,

There's a short section on [this page](http://cfconventions.org/conventions.html) headed with: "Conventions preceding CF 1.0".  There we link to documents describing COARDS and GDT, but there is no link to the attached document ([Eaton_conventions_draft2.pdf](https://github.com/user-attachments/files/18070010/Eaton_conventions_draft2.pdf)) that Brian Eaton found for me somewhere in his archives.  

I suggest we either:

1. host his html version of this document (which I have but was unable to attach) on our CF site and link to it from the page above where we currently reference "NCAR CSM", or
2. pass Brian's document through zenodo with an appropriate author list.

Brian said he would 
> be willing to put in the effort to put this on zenodo if you think
> there's a compelling reason to do so.  But I'm also satisfied to simply
> have it preserved on the cfconventions site.  Let me know your thoughts.

Perhaps we should post it as is on our site, and if anyone thinks it's a good idea to zenodo it, we should ask Brian to do that.

best regards,
Karl

**Comment by ChrisBarker-NOAA** [comment]
I think a copy on the CF site is fine.

The more time passes, the less important the precursors are ...

Brian should only Zenodo it if he wants to for his own reasons.

**Comment by larsbarring** [comment]
In fact, I was in https://github.com/cf-convention/cf-convention.github.io/issues/569 suggesting that we should move in the direction of making this kind of documents available on zenodo. 

For us, right now when we have the focus on enhancing and developing CF, and of course making use of CF, it may not be immediately relevant to have such documents published. But there is an outside world that may look at CF from many different angles, such as its historic development and what impact it may have had on the the discourse on climate and climate change, as well as its relevance for ensemble based analyses, and what not.

A very different angle is the ongoing work trying to curb the ever-growing size of the website repo size and clean up its underlying directory structure.

That's why I opened #569, but I do not have a strong opinion in either direction. And, of course, it ultimately is up to the document author/owner to decide what to do.

**Comment by ethanrd** [comment]
I agree with Lars that there will likely be interest in this and other such documents that aren't currently apparent to us. I often review the history of projects when writing reports and proposals and it is frustrating when I can't find a document I know existed. Also, I would say that web sites are notoriously bad at maintaining historical documents.

So, one vote here for getting it into zenodo. @brian-eaton Happy to help out. Let me know if you have any questions or anything.

**Comment by JonathanGregory** [comment]
Thanks, Brian and Karl @taylor13, for the PDF of the NCAS-CSM convention. I have prepared PR #571 to link it to the [conventions page](https://cfconventions.org/conventions.html), where it's already mentioned, in the last section of the page, along with COARDS and GDT. Does anyone disagree with our hosting this copy? All these files are small, and not a concern for the size of the repo or the Pages website.

I think it's good to put it in Zenodo as well, also like COARDS and GDT.

**Comment by brian-eaton** [comment]
Thanks for all the feedback.  I'll follow up with @ethanrd  for help getting it into zenodo.

**Comment by larsbarring** [comment]
Dear Jonathan @JonathanGregory,  I do not disagree (not at all!) with our hosting a copy of this document, or any of the other historic documents. But if we also have them on zenodo, which I do think is a good thing as mentioned above, I do think it is *unnecessary*. We should of course continue to link to them on the webpage as we do as today,  but the link will point to the zenodo document rather than a file in the local directory structure (somewhere under .../Data/...).  

Literally as I wrote this Karl's comment arrived, and I agree with this.

**Comment by larsbarring** [comment]
If we are thinking about this from a safety perspective, I think the least we have to worry about is what we have on zenodo. This is from [their web page on policies](https://about.zenodo.org/policies/):

> Longevity
>
>   * Versions: Data files are versioned. Records are not versioned. The uploaded data is archived as a Submission Information Package. Derivatives of data files are generated, but original content is never modified. Records can be retracted from public view; however, the data files and record are preserved.
>   * Replicas: All data files are stored in CERN Data Centres, primarily Geneva, with replicas in Budapest. Data files are kept in multiple replicas in a distributed file system, which is backed up to tape on a nightly basis.
>   * Retention period: Items will be retained for the lifetime of the repository. This is currently the lifetime of the host laboratory CERN, which currently has an experimental programme defined for the next 20 years at least.
>   * Functional preservation: Zenodo makes no promises of usability and understandability of deposited objects over time.
>   * File preservation: Data files and metadata are backed up nightly and replicated into multiple copies in the online system.
>   * Fixity and authenticity: All data files are stored along with a MD5 checksum of the file content. Files are regularly checked against their checksums to assure that file content remains constant.
>   * Succession plans: In case of closure of the repository, best efforts will be made to integrate all content into suitable alternative institutional and/or subject based repositories.

**Comment by ethanrd** [comment]
A quick update:
The NCAR-CSM NetCDF Conventions document has been uploaded to Zenodo. The DOI is https://doi.org/10.5281/zenodo.14649053.
Thanks @brian-eaton!

**Comment by JonathanGregory** [comment]
Thanks from me too, @brian-eaton.

I have updated my PR #571 to link the NCAR-CSM convention from the section of the [Conventions page](https://cfconventions.org/conventions.html) about conventions before CF 1.0. I have linked both the Zenodo DOI and a copy of the PDF in CF GitHub. I've also changed the GDT text so it links both the Zenodo DOI (which Paul Durack and Karl organised) and the local copies we already had.

I understand that Zenodo is extremely reliable, as described by Lars above. I don't fear they will lose or mess up our files! However, I think it's worthwhile to have copies on the CF website, for these reasons: (a) If you go to Zenodo, you get a landing page, which is certainly informative, but it's one more click if you want to view the PDF in the whole browser window; (b) for GDT, we have HTML as well as PDF; (c) although Zenodo is safe, it seems conceivable that you might temporarily not be able to connect to it, since the internet is not perfectly reliable. Is there a reason _not_ to have an identical copy? They're small files and they're not going to change so no danger of inconsistency.

Cheers, Jonathan
